api/src/iot/import_utils.py

```diff
@@ -746,6 +751,15 @@ def import_sensor(sensor_data: SensorData, owner: models.Person2, action_logger=
         theme = models.Theme.objects.get_or_create(name=theme_name)[0]
         device.themes.add(theme)
 
+    for observation_goal in sensor_data.observation_goals:
+        import_result = models.ObservationGoal.objects.get_or_create(
```

Comment:
Kun je dit wrappen in de `action_logger` functie zodat we een historie opbouwen van deze acties?